pages/pipelines/environment_variables.md.erb

```diff
@@ -122,18 +202,30 @@ The following environment variables are automatically provided to every job, and
     <th><code>BUILDKITE_REBUILT_FROM_BUILD_NUMBER</code></th>
     <td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"1514"</code>, or <code>""</code> if not a rebuild.</p></td>
   </tr>
+  <tr>
+    <th><code>BUILDKITE_REFSPEC</code></th>
```

Comment:
I think we already have this one in the list?

Reply:
It was out of alphabetical order just slightly, I moved it into it's correct spot

Reply:
Ahhhh!

Reply:
Ahhhh!